servo/connectors/kubernetes.py

```diff
@@ -4931,6 +5184,10 @@ class OptimizationStrategy(str, enum.Enum):
     adjustments to it instead of the Deployment itself.
     """
 
+    none = "none"
+    """The none strategy implements bare-bones servo infrastructure for monitoring only, without native adjustments.
```

Comment:
Please address for https://github.com/opsani/servox/blob/0fbbd90c146cba357d2f5060585c719bd988336d/servo/connectors/kubernetes.py#L5048

(My guess @ekalosak is that its native to the kubernetes environment although it makes more sense in [some contexts](https://github.com/opsani/servox/blob/0fbbd90c146cba357d2f5060585c719bd988336d/servo/connectors/kubernetes.py#L3411) than others)

Reply:
@ekalosak What I had meant originally by native (with a liberal use of that definition) was by base servox vs. external connectors. Nullified after your clarification that HPA isn't really an external connector and also this line being removed regardless :) 

I used the term once elsewhere in defining the `create_tuning_pod` property which I have reworded for clarity simply as:
"Disable to prevent a canary strategy with tuning pod adjustments"